client/plugins/screenlock.py

```diff
@@ -24,7 +24,7 @@
 valid_lockers = (
     'betterlockscreen', 'slock', 'i3lock', 'i3lock-fancy', 'i3lock-fancy-rapid'
 )
-valid_lockers_re = re.compile(r'^(?:' +
+valid_lockers_re = re.compile(r'(?:' +
```

Comment:
is this to allow absolute paths?

Reply:
Yes, that's correct.

Reply:
Can we use `(?:^|/)`, or perhaps replace `^` with `\b`, instead of getting rid of the `^` entirely. I would prefer the regexp didn't match mid-word.

Reply:
@jikamens done. Thanks.

Reply:
![image](https://user-images.githubusercontent.com/1739649/59054155-16f6f780-8861-11e9-8778-2b912118768a.png)